docker/datahub-gms/env/docker.env

```diff
@@ -2,18 +2,14 @@ DATASET_ENABLE_SCSI=false
 EBEAN_DATASOURCE_USERNAME=datahub
 EBEAN_DATASOURCE_PASSWORD=datahub
 EBEAN_DATASOURCE_HOST=mysql:3306
-EBEAN_DATASOURCE_URL=jdbc:mysql://mysql:3306/datahub?verifyServerCertificate=false&useSSL=true&useUnicode=yes&characterEncoding=UTF-8&enabledTLSProtocols=TLSv1.2
+EBEAN_DATASOURCE_URL=jdbc:mysql://mysql:3306/datahub?verifyServerCertificate=false&useSSL=true&useUnicode=yes&characterEncoding=UTF-8
 EBEAN_DATASOURCE_DRIVER=com.mysql.jdbc.Driver
 KAFKA_BOOTSTRAP_SERVER=broker:29092
 KAFKA_SCHEMAREGISTRY_URL=http://schema-registry:8081
 ELASTICSEARCH_HOST=elasticsearch
 ELASTICSEARCH_PORT=9200
-NEO4J_HOST=http://neo4j:7474
-NEO4J_URI=bolt://neo4j
-NEO4J_USERNAME=neo4j
-NEO4J_PASSWORD=datahub
+GRAPH_SERVICE_IMPL=elasticsearch
 JAVA_OPTS=-Xms1g -Xmx1g
-GRAPH_SERVICE_IMPL=neo4j
```

Comment:
is changing this default going to affect any ongoing deployments?

Reply:
right, this looks like a breaking change as this is not quickstart specific but more general

I could use symlinks here as I presume these files are not fetched via http, then the naming and default becomes more obvious and I will revert the `docker.env` to neo4j.

Reply:
@EnricoMi is this related to the other changes being made in this PR? I wonder if we can just revert this and then bring it back up as an individual change. Switching the default graph service is going to be a little dicey and i would want that pr to be as slim as possible

Reply:
working on reverting this so that neo4j is the default again